MediaBrowser.Common/Net/DefaultHttpClient.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace MediaBrowser.Common.Net
+{
+    /// <summary>
+    /// Default http client.
+    /// </summary>
+    public class DefaultHttpClient
```

Comment:
Why is this needed?

Reply:
Allows automatic handler usage, and I believe better practice.
Eventually we should move to specific HttpClients but this will allow the remaining IHttpClient removals